src/Form/index.tsx

```diff
@@ -845,7 +847,7 @@ function Form({
       return { key, [(val as any).type]: newVal };
     });
     const stepPromise =
-      featheryFields.length > 0
+      featheryFields.length > 0 && !_draft
```

Comment:
maybe it would be cleaner to initialize the client with the draft boolean and check the flag on the write calls within the client functions